### DISCUSSION TITLE
Is there any way to localize Strings in extension

### DISCUSSION THREAD

**Discussion by Kwaadpepper** [Ulauncher/Ulauncher#899] [Extensions and Themes]
## Question
All is in the title. I saw nothing in the docs.

EDIT: just found something I am not used to python, I used gettext package.
https://github.com/Kwaadpepper/zealdoc/blob/master/lib/i18n.py

But still I think now there is no way to localize manifest.json, since preferences are definitely read only from extensions.
Will you provide a way to localize "name" and "description" attributes from manifest.json ?

**Comment by friday** [comment]
I wouldn't know how to handle or prioritize this. I don't think it would pick up much use. Supporting localization is more work for extension developers than it is putting together most extension.
